src/components/Sections/SectionHeader.tsx

```diff
@@ -1,50 +1,66 @@
 import React from 'react'
+import cx from 'classnames'
 
 import ActionsMenu from 'cozy-ui/transpiled/react/ActionsMenu'
 import Button from 'cozy-ui/transpiled/react/Buttons'
+import CozyTheme from 'cozy-ui/transpiled/react/providers/CozyTheme'
 import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
 import DotsIcon from 'cozy-ui/transpiled/react/Icons/Dots'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import flag from 'cozy-flags'
+import { makeStyles } from 'cozy-ui/transpiled/react/styles'
 
-import { actions } from 'components/Sections/SectionActions'
 import { SectionHeaderProps } from 'components/Sections/SectionsTypes'
+import { actions } from 'components/Sections/SectionActions'
+
+const useStyles = makeStyles(() => ({
+  headerWrapper: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '100%'
+  }
+}))
 
 export const SectionHeader = ({
-  name,
+  section,
   anchorRef,
   toggleMenu,
-  menuState,
-  handleAction
-}: SectionHeaderProps): JSX.Element => (
-  <>
-    <Divider className="u-mv-0" variant="subtitle2">
-      {name}
-    </Divider>
+  menuState
+}: SectionHeaderProps): JSX.Element => {
+  const classes = useStyles()
 
-    {flag('home.detailed_section.show_more-dev') && (
-      <>
-        <Button
-          label={<Icon icon={DotsIcon} />}
-          ref={anchorRef}
-          onClick={toggleMenu}
-        />
+  return (
+    <>
+      <div className={cx([classes.headerWrapper])}>
```

Comment:
Instead using makeStyles, we try to use utility classes. I think you can use `u-flex u-w-100 u-flex-justify-between` in this case ([doc](https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html))

Reply:
Updated ✅